### PR TITLE
[Node.js] Add client sample and release scripts

### PR DESCRIPTION
**Pull request by noodlefrenzy**
Just for reviewing purposes

### CODE REVIEW

File: build/release/release_npm.cmd [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+@setlocal EnableDelayedExpansion
+@echo off
+
+cd %~dp0\release_npm
+set root=../../..
+
+@REM encourage user to specify a fork of azure-event-hubs
+if "%1"=="" goto usage
+for /f "tokens=1,2,3" %%i in ('git remote -v') do (
```
Comment (noodlefrenzy) [inline build/release/release_npm.cmd:12]: I'm going to assume you know what you're doing here, as my brain is getting a parse error.

File: node/samples/simple_eventhubs_client.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  console.log('Event Received: ');
+  console.log(JSON.stringify(ehEvent.body));
+  console.log('');
+};
+
+var client = EventHubClient.fromConnectionString(connectionString, eventHubPath);
+
+client.open()
+      .then(client.getPartitionIds.bind(client))
+      .then(function (partitionIds) {
```
Comment (noodlefrenzy) [inline node/samples/simple_eventhubs_client.js:31]: could use .map here.

**Comment by noodlefrenzy** [comment]
So AFAICT this is tarballing all packages in their own named/versioned staging branches, and _not_ actually publishing to npm, yes?

**Comment by pierreca** [comment]
that is correct. only generating the tarball. pushing to npm is done in a dedicated build job

**Comment by noodlefrenzy** [comment]
Cool, then, :shipit:

**Pull request merged**